components/crud-web-apps/jupyter/frontend/src/app/pages/form/form-default/form-image/form-image.component.html

```diff
@@ -4,8 +4,23 @@
   ML packages"
   icon="fa:fab:docker"
 >
+  <section>
+    <div class="server-type">Choose server type:</div>
```

Comment:
I propose to remove this text and only have the buttons. It will make the form more clean